packages/datagateway-table/src/dls/detailsPanels/datasetDetailsPanel.component.tsx

```diff
@@ -73,11 +74,13 @@ const DatasetDetailsPanel = (
           ) : (
             <Button
               onClick={() => {
-                // TODO
+                const { fetchSize } = props;
```

Comment:
@tswinb you can probably add this at the top as well along with the other props.